### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by AlanRuijia**
Correct some minor mistakes in README and make it better

A GIF or MEME to give some spice of the internet

## Motivation and Context

[issue](https://github.com/BrainJS/brain.js/issues/###)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Author's Checklist:

- [ ] My code focuses on the main motivation and avoids scope creep.
- [ ] My code passes current tests and adds new tests where possible.
- [ ] My code is [SOLID](https://en.wikipedia.org/wiki/SOLID_(object-oriented_design)) and [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).
- [ ] I have updated the documentation as needed.

## Reviewer's Checklist:
- [ ] I kept my comments to the author positive, specific, and productive.
- [ ] I tested the code and didn't find any new problems.
- [ ] I think the motivation is good for the project.
- [ ] I think the code works to satisfies the motivation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,7 +100,7 @@ You can still download the latest, though. They are cool!
 Download the latest [brain.js for browser](https://raw.githubusercontent.com/harthur-org/brain.js/master/browser.js). Training is computationally expensive, so you should try to train the network offline (or on a Worker) and use the `toFunction()` or `toJSON()` options to plug the pre-trained network into your website.
 
 # Training
-Use `train()` to train the network with an array of training data. The network has to be trained with all the data in bulk in one call to `train()`. The more training patterns, the longer it will probably take to train, but the better the network will be at classifying new patterns.
+Use `train()` to train the network with an array of training data. The network has to be trained with all the data in bulk in one call to `train()`. The more training patterns, the longer it probably takes to train, and the better the network will be at classifying new patterns.
```
Comment (freddyC) [inline README.md]:
```
The more training patterns, the longer it probably takes to train, and the better the network will
be at classifying new patterns.
```

Kinda sounds funny.  What do you think of this (or something like it)?

```
More training patterns will probably take longer to train, but will usually result in a network better
at classifying new patterns.
```

**Review by freddyC — APPROVED** [review]
It looks real good.  I have one comment, but it's just a suggestion.  
giphy

File: README.md (hunk 1 above)
Comment (AlanRuijia) [reply README.md]: I think the latter one is better.

**Comment by robertleeplummerjr** [comment]
Nice work.

**Pull request merged**